templates/templates/_navigation.html

```diff
@@ -186,14 +186,24 @@ <h5 class="p-navigation--secondary__logo">
 
   function openSearch(e) {
     e.preventDefault();
+    var globalNav = document.querySelector('.global-nav__row');
+    var globalNavDropdown = document.querySelector('.global-nav__dropdown-content');
     var navigation = document.querySelector('.p-navigation__nav');
     var dropdownWindowOverlay = document.querySelector(".dropdown-window-overlay");
+    var banner = document.querySelector(".p-navigation__banner");
+    var dropdownWindow = document.querySelector(".dropdown-window");
+    var navigationItems = document.querySelector('.p-navigation__items');
     var searchButton = document.querySelector('.js-search-button');
     var search = document.querySelector('.p-navigation__search');
     var searchInput = document.querySelector('.p-search-box__input');
     var searchActive = !search.classList.contains('u-hide');
     search.classList.remove('u-hide');
     searchButton.classList.add('u-hide');
+    globalNav.style= "max-width:68rem; opacity: 0.4; transition: opacity 0.5s ease-in-out;"
```

Comment:
Lets not do this as the expected place to set the max-width is the init script of the global nav. I think its better to leave it untouched and active when searching.